tests/e2e/utils/activate-amp-and-set-mode.js

```diff
@@ -0,0 +1,40 @@
+
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * The allow list of AMP modes.
+ *
+ */
+export const allowedAmpModes = {
+	standard: 'standard',
+	transitional: 'transitional',
+	reader: 'disabled',
+};
+
+/**
+ * Activate AMP and set it to the correct mode.
+ *
+ * @param {string} mode The mode to set AMP to. Possible value of standard, transitional or reader.
+ */
+export const activateAmpAndSetMode = async ( mode ) => {
+	await activatePlugin( 'amp' );
+	await setAMPMode( mode );
+};
+
+/**
+ * Set AMP Mode
+ *
+ * @param {string} mode The mode to set AMP to. Possible value of standard, transitional or reader.
+ */
+export const setAMPMode = async ( mode ) => {
+	// Test to be sure that the passed mode is known.
+	await expect( Object.keys( allowedAmpModes ) ).toContain( mode );
```

Comment:
No need to `await` here as this is synchronous vanilla Jest 👍 

Also, I think we can use `expect( allowedAMPModes ).toHaveProperty( mode )`
https://jestjs.io/docs/en/expect#tohavepropertykeypath-value

Reply:
ah nice - that's a much more concise way of checking.